hack/clean-e2e.sh

```diff
@@ -4,8 +4,12 @@ REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 cd "${REPO_ROOT}" || exit 1
 
 minikube delete
+docker stop vbmc
```

Comment:
Minor issue: this step seems to be redundant. `docker rm -f` should stop and clean the container if it's running. Otoh `docker stop` will give out an error if the container is not running, which means we will always have some weird errors unless we run both the emulators.

Reply:
Ok true, I can't remember how they got in here but I think you are right. Let's remove them.